addon/index.js

```diff
@@ -138,5 +142,15 @@ import Row from './classes/Row';
  * @module Usage
  * @submodule Component Declaration
  */
+
 export default Table;
-export { Table, Column, Row };
+
+export {
+  Table,
+  Column,
+  Row,
+  FocusBehavior,
```

Comment:
I don't think we need to export the behaviors. Who ever needs them can grab them from the behavior dir.